shell/src/main/java/alluxio/cli/fs/command/PersistCommand.java

```diff
@@ -47,71 +68,116 @@ public String getCommandName() {
     return "persist";
   }
 
+  @Override
+  public Options getOptions() {
+    return new Options().addOption(PARALLELISM_OPTION);
+  }
+
   @Override
   public void validateArgs(CommandLine cl) throws InvalidArgumentException {
     CommandUtils.checkNumOfArgsNoLessThan(this, cl, 1);
   }
 
   @Override
-  protected void runPlainPath(AlluxioURI plainPath, CommandLine cl)
-      throws AlluxioException, IOException {
-    persist(plainPath);
+  public String getUsage() {
+    return "persist [-p|--parallelism <#>] <path> [<path> ...]";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Persists files or directories currently stored only in Alluxio to the UnderFileSystem.";
   }
 
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
+    int parallelism = 4;
```

Comment:
make this constant and use it to document the default value?

Reply:
Updated, thanks.